adapters/repos/db/replication.go

```diff
@@ -378,6 +385,13 @@ func (i *Index) IncomingOverwriteObjects(ctx context.Context,
 	return i.overwriteObjects(ctx, shardName, vobjects)
 }
 
+func (db *DB) DigestObjects(ctx context.Context,
+	class, shardName string, ids []strfmt.UUID,
+) (result []replica.RepairResponse, err error) {
+	index := db.GetIndex(schema.ClassName(class))
+	return index.digestObjects(ctx, shardName, ids)
+}
+
```

Comment:
I think we are missing the deletion flag in the digest read request.
See `digestObjects(`) lines 415-419
```
result[j] = replica.RepairResponse{
			ID:         objs[j].ID().String(),
			UpdateTime: objs[j].LastUpdateTimeUnix(),
                         Deleted : true|false, // TODO
			// TODO: use version when supported
			Version: 0,
		}
```

Reply:
fixed